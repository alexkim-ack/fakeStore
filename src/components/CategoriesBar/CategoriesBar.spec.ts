import { expect, describe, it } from "vitest";

import { CategoriesBar } from "./index";

const test: string[] = [
    "electronics",
    "jewelery",
    "men's clothing",
    "women's clothing",
];

describe("CategoriesBar", () => {
    it("should render", () => {
        const render = CategoriesBar({ categories: test, callBack: () => {} });
        expect(render).toMatchSnapshot();
    });
});
