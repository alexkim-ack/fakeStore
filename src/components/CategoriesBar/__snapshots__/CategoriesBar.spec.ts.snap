// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CategoriesBar > should render 1`] = `
<ul
  className="fixed top-[88px] left-[0px]
                w-full flex flex-row items-between
                border-green-500
                bg-green-500 list-none"
>
  <li
    className="p-[6px] hover:bg-green-700 rounded-md"
    onClick={[Function]}
  >
    <span
      className="text-white"
    >
      electronics
    </span>
  </li>
  <li
    className="p-[6px] hover:bg-green-700 rounded-md"
    onClick={[Function]}
  >
    <span
      className="text-white"
    >
      jewelery
    </span>
  </li>
  <li
    className="p-[6px] hover:bg-green-700 rounded-md"
    onClick={[Function]}
  >
    <span
      className="text-white"
    >
      men's clothing
    </span>
  </li>
  <li
    className="p-[6px] hover:bg-green-700 rounded-md"
    onClick={[Function]}
  >
    <span
      className="text-white"
    >
      women's clothing
    </span>
  </li>
</ul>
`;
